### PR TITLE
fix: make url more obvious

### LINKED ISSUE #530: bug - style: make NOW URL more obvious
It takes some looking to find the hosted NOW URL after everything is set up. It should be more prominent. The first thing I see if the "Flow is available!" box

image

### PR DESCRIPTION
**Pull request by azayz**
closes #530

### CODE REVIEW

**Comment by azayz** [comment]
New print

Screenshot from 2022-09-27 10-24-15

File: now/run_all_k8s.py [hunk 1]
```diff
@@ -141,8 +141,12 @@ def start_now(app_instance, **kwargs):
         + (f'&port={gateway_port_internal}' if gateway_port_internal else '')
     )
     print()
-    print(f'BFF docs are accessible at:\n{bff_url}')
-    print(f'Playground is accessible at:\n{playground_url}')
+    print(
```
Comment (makram93) [inline now/run_all_k8s.py:144]: Highlighting is good. Can we also make it in a tabular format like the JCloud print?

|    BFF  docs  |  ...          |
| ------------- |:-------------:| 
| Playground    | ... (the URL should be wrapped in this column |

**Comment by azayz** [comment]
Screenshot from 2022-09-27 14-46-33

**Review by joschkabraun — COMMENTED** [review]
I prefer the table solution / version.

**Comment by azayz** [comment]
Screenshot from 2022-09-27 15-08-49

File: now/run_all_k8s.py [hunk 2]
```diff
@@ -141,8 +145,15 @@ def start_now(app_instance, **kwargs):
...
-    print(f'BFF docs are accessible at:\n{bff_url}')
-    print(f'Playground is accessible at:\n{playground_url}')
+    my_table = Table(
+        'Attribute', 'Value', show_header=False, box=box.SIMPLE, highlight=True
+    )
+    my_table.add_row('BFF docs', bff_url)
+    my_table.add_row('Playground', playground_url)
+    console = Console()
+    console.print(
+        Panel(my_table, title=':tada: Bff & Playground are accessible!', expand=False)
```
Comment (jupyterjazz) [inline now/run_all_k8s.py]:
```suggestion
        Panel(my_table, title=':tada: BFF & Playground are accessible!', expand=False)
```

**Review by jupyterjazz — APPROVED** [review]
LGTM!

**Review by joschkabraun — APPROVED** [review]
LGTM!

**Pull request merged**